app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    person_list = []
+
+    for person_data in people:
+        person = Person(name=person_data["name"], age=person_data["age"])
+        person_list.append(person)
+
+    for person_data in people:
+        person = Person.people[person_data["name"]]
+        if "wife" in person_data and person_data["wife"]:
+            person.wife = Person.people[person_data["wife"]]
+        elif "husband" in person_data and person_data["husband"]:
```

Comment:
Use `person_data.get('husband')` instead of checking with `in` to simplify the code and avoid potential `KeyError` exceptions, as suggested in the checklist.